src/en/server-hosting/popular-forks-cdns.md

```diff
@@ -0,0 +1,46 @@
+# List of codebases with prebuilt server packages
+
+This document will serve as a list of forks known to have a prebuilt server software for players to download.
+
+This doc was last updated at 2024-10-01 13:52:04 UTC
```

Comment:
Probably don't need to specify the time of day too.

Reply:
PRECISION